src/components/accounts/show.js

```diff
@@ -50,19 +50,19 @@ class AccountRender extends React.Component {
         };
     }
 
-    handleEdit = () => {
+    handleEdit() {
```

Comment:
these should all be arrow functions, or else they need `this.bind` in the constructor. otherwise it is not bound to `this`

(that's why @whilei had  problem with `Cannot read property 'setState' of null`)